snuba/datasets/processors/spans_processor.py

```diff
@@ -27,7 +27,7 @@
 )
 from snuba.utils.metrics.wrapper import MetricsWrapper
 
-logger = logging.getLogger(__name__)
+logger = structlog.get_logger(__name__)
```

Comment:
Have we used structlog in any of our consumer processors before? Not sure if there is a perf impact of using it or not. Do you happen to know?

Reply:
We use it subscriptions. Not sure if we've have done performance testing on ti.